src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAccountController.cs

```diff
@@ -49,6 +52,7 @@ public class EmployerAccountController : BaseController
             _returnUrlCookieStorageService = returnUrlCookieStorageService;
             _accountCookieStorage = accountCookieStorage;
             _hashedAccountIdCookieName = typeof(HashedAccountIdModel).FullName;
+            _authorizationService = authorizationService;

```

Comment:
We are only injecting the dependency on `IAuthorizatoin` servcie to set a property on a model which we than pass to the _next layer down_, this seems a bit redudant. See other comment on lack of test coverage on property setting in this controller.